### PR TITLE
Update API master with new APIs

### PR DESCRIPTION
**Pull request by grasskin**
Still some saving APIs that I wasn't sure where to add but they can be a part of a second round. 

Unclear on the actual process to generate the site once we update API_MASTER.

### CODE REVIEW

File: scripts/api_master.py [hunk 1]
```diff
@@ -116,6 +116,7 @@
                         "keras.Model.get_config",
                         "keras.layers.Layer.add_loss",
                         "keras.layers.Layer.losses",
+                        "keras.layers.Wrapper",
```
Comment (fchollet) [inline scripts/api_master.py:119]: You can remove this one

File: scripts/api_master.py [hunk 2]
```diff
@@ -189,11 +195,26 @@
                             "title": "Input object",
                             "generate": ["keras.Input"],
                         },
+                        {
+                            "path": "input_layer",
+                            "title": "Input layer",
+                            "generate": ["keras.layers.InputLayer"],
```
Comment (fchollet) [inline scripts/api_master.py:201]: You can remove this one. It doesn't even have a docstring.

File: scripts/api_master.py [hunk 3]
```diff
@@ -840,6 +912,21 @@
                     "title": "ProgbarLogger",
                     "generate": ["keras.callbacks.ProgbarLogger"],
                 },
+                {
+                    "path": "history_callback",
+                    "title": "History",
+                    "generate": ["keras.callbacks.History"],
```
Comment (fchollet) [inline scripts/api_master.py:918]: You can remove this one

File: scripts/api_master.py [hunk 4]
```diff
@@ -840,6 +912,21 @@
...
+                },
+                {
+                    "path":"swap_ema_weights", 
+                    "title": "SwapEMAWeights",
+                    "generate": ["keras.callbacks.SwapEMAWeights"]
+                },
+                {
+                    "path": "callback_list",
+                    "title": "CallbackList",
+                    "generate": ["keras.callbacks.CallbackList"]
```
Comment (fchollet) [inline scripts/api_master.py:928]: You can also remove this one

File: scripts/api_master.py [hunk 5]
```diff
@@ -874,6 +961,7 @@
                         "keras.ops.bincount",
                         "keras.ops.broadcast_to",
                         "keras.ops.ceil",
+                        "keras.ops.cholesky"
```
Comment (fchollet) [inline scripts/api_master.py:964]: We need to create a new category for linalg ops, with all ops from `keras.ops.linalg`.

File: scripts/api_master.py [hunk 6]
```diff
@@ -1188,6 +1296,13 @@
             "title": "Metrics",
             "toc": True,
             "children": [
+                {
+                    "path": "base_metric",
+                    "title": "The base Metric class",
```
Comment (fchollet) [inline scripts/api_master.py:1301]: Just "Base Metric class", it's cleaner.

File: scripts/api_master.py [hunk 7]
```diff
@@ -1618,30 +1756,53 @@
                         "keras.utils.FeatureSpace",
                     ],
                 },
+                {
+                    "path": "tensor_utils",
+                    "title": "Tensor graph utilities",
```
Comment (fchollet) [inline scripts/api_master.py:1761]: Drop the "graph"

File: scripts/api_master.py [hunk 8]
```diff
@@ -1324,6 +1460,8 @@
                         "keras.utils.load_img",
                         "keras.utils.img_to_array",
                         "keras.utils.save_img",
+                        "keras.preprocessing.image.smart_resize",
```
Comment (fchollet) [inline scripts/api_master.py:1463]: You can drop this one, it's legacy

**Review by fchollet — COMMENTED** [review]
Thanks for the PR!

File: scripts/api_master.py (hunk 5 above)
Comment (grasskin) [reply scripts/api_master.py]: Fixed! Moved all keras.ops.linalg ops to new section and removed them from core/numpy

**Review by fchollet — APPROVED** [review]
LGTM, thank you!
